Formula/ydk.rb

```diff
@@ -0,0 +1,43 @@
+class Ydk < Formula
+  desc "generate API bindings to YANG data models"
+  homepage "https://github.com/CiscoDevNet/ydk-cpp"
+  url "https://github.com/CiscoDevNet/ydk-cpp/archive/0.5.2.tar.gz"
+  sha256 "39ca26b57e0d784243ebd0c07eb0e35fc0ad8600886fde2be4440eae898b844d"
+
+  depends_on "cmake" => :build
+  depends_on "boost"
+  depends_on "boost-python"
+  depends_on "curl"
+  depends_on "libssh"
+  depends_on "pcre"
+  depends_on "xml2"
+  depends_on "pkg-config" => :build
+  depends_on :x11 => :optional
```

Comment:
Let's remove this, we're trying to reduce options and particularly X11 ones, thanks.